controllers/imageController.js

```diff
@@ -69,6 +72,26 @@ ImageController.prototype.setUserAvatar = function(req, res, next) {
 	})
 }
 
+
+ImageController.prototype.setUserAvatar = function(req, res, next) {
+	var processor = new ImageProcessor(this._fs)
+	return this._setUserProfileImage(req, res, next,
+		processor.handleUserAvatarUpload.bind(processor),
+		'/' + req.user.username + '/profile',
+		{original: 'avatarOriginal', scaled: 'avatarScaled'}
+	)
+}
+
+ImageController.prototype.setUserHeader = function(req, res, next) {
+	var processor = new ImageProcessor(this._fs)
+	return this._setUserProfileImage(req, res, next,
+		processor.handleUserHeaderUpload.bind(processor),
+		'/' + req.user.username + '/header',
```

Comment:
these will change, it's too early to say how. I'm more inclined to free up /profile entirely instead, but this is an interim PR so would like to keep things for now. note this is unused in the UI yet.